tests/integration/pom.xml

```diff
@@ -12,20 +12,30 @@
 	<description>Tests which ensure that Mule sub projects work well together.</description>
 	<packaging>jar</packaging>
 
-    <properties>
-        <skipExportTests>false</skipExportTests>
-        <skipTests>${skipIntegrationTests}</skipTests>
-    </properties>
+	<properties>
```

Comment:
formatting of this XML file does not seems correct. We use 4 chars indentation, not 8.